src/main/java/seedu/address/logic/commands/WhitelistCommand.java

```diff
@@ -15,16 +15,20 @@
 import seedu.address.model.person.Person;
 
 /**
- * Whitelists and existing person.
+ * Whitelists a previously-blacklisted person.
  */
 public class WhitelistCommand extends Command {
 
     public static final String COMMAND_WORD = "whitelist";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + " index cs/CLIENT_STATUS"
-            + ": removes person from the blacklist";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " INDEX cs/NEW_CLIENT_STATUS"
```

Comment:
Most of the user guide capitalizes "INDEX," but the whitelist and blacklist section uses lowercase "index."  Perhaps we should standardize this to "INDEX" for consistency.